cogs/fun.py

```diff
@@ -48,6 +48,13 @@ async def api_img_creator(self, ctx, url: str, filename: str, content: str = Non
             bio = BytesIO(req)
             bio.seek(0)
             await ctx.send(content=content, file=discord.File(bio, filename=filename))
+    
+    @commands.command(aliases=['give_meme'])       
```

Comment:
Since we are using Doxygen, it would be a good idea to put a Doxygen comment above this. Would want it to say what this is and what user story it is related too.